karma.conf.js

```diff
@@ -25,6 +25,9 @@ module.exports = (config) => {
             ],
             esm: {
                 nodeResolve: true,
+                babelConfig: {
+                    plugins: ['transform-node-env-inline'],
+                },
```

Comment:
No need to do the work here, but can you create an issue to merge the ESM configs between here at `.storybook/main.js`?